cmd/kubernetesDeploy.go

```diff
@@ -38,7 +39,20 @@ func runKubernetesDeploy(config kubernetesDeployOptions, telemetryData *telemetr
 	telemetryData.Custom1 = config.DeployTool
 
 	if config.DeployTool == "helm" || config.DeployTool == "helm3" {
-		return runHelmDeploy(config, utils, stdout)
+		err := runHelmDeploy(config, utils, stdout)
+		// download and execute teardown script
+		if len(config.TeardownScript) > 0 {
```

Comment:
Late to the party, but did we really have to invent another extension mechanism besides the standard Piper extensibility feature? @OliverNocon